### ISSUE TITLE
course view broken

### ISSUE THREAD

**Issue by SimonKoehl** [serlo-org/athene2#683]
https://de.serlo.org/community/neue-faecher-themen/medienbildung/datensammlung-big-data-ist-big-data

This happens with every course from https://de.serlo.org/community/neue-faecher-themen/medienbildung/datensammlung-big-data

Other courses on serlo.org seem to be fine: https://de.serlo.org/61911/uebersicht

![course view broken](https://user-images.githubusercontent.com/3379377/40252344-64822b86-5adc-11e8-997f-876bac1e3088.PNG)

**Comment by Knorrke** [comment]
Courses don't have a view, they redirect to their first child (course-page). If there isn't any page with an accepted revision, then this message is shown

**Comment by SimonKoehl** [comment]
Is this string used for courses only? Because then I can formulated it more specific.

**Comment by SimonKoehl** [comment]
and/or this can be solved via #680

**Comment by Knorrke** [comment]
Well, it's a generic message, but you usually only encounter it in courses atm. But if we want to have a special view for empty courses, then I'd just add that as view for the course. That's not a problem.

**Comment by inyono** [comment]
IMHO: #812 resolves this issue, too

**Comment by SimonKoehl** [comment]
I don't think so @inyono because also with #812 beeing implemented, we can have the situation of a course without a first course page. Don't we?

**Comment by inyono** [comment]
#812 would still handle courses w/o a course page. It would always have a view for the course (i.e. an empty page / fallback text when no course page exists).
